go.mod

```diff
@@ -34,119 +34,10 @@ require (
 	google.golang.org/protobuf v1.27.1
 )
 
-require (
-	filippo.io/edwards25519 v1.0.0-beta.2 // indirect
```

Comment:
This was all unnecessary??

Reply:
I think there are some issues in the team with using different golang versions. When 1.18 is released we should all switch to that